archunit/src/main/java/com/tngtech/archunit/core/domain/JavaAccess.java

```diff
@@ -107,6 +107,14 @@ String additionalToStringFields() {
         return "";
     }
 
+    /**
+     * @return all matching class dependencies.
+     */
+    @PublicAPI(usage = ACCESS)
+    public Set<Dependency> toClassDependencies() {
```

Comment:
The name might be inspired by the previous usage of `Dependency.tryCreateFromAccess` in `JavaClassDependencies` (which is no public API). I wonder whether the name `getDependencies` might be more intuitive?

Reply:
The method name was chosen as in the existing method com.tngtech.archunit.library.modules.ModuleDependency#toClassDependencies

To be honest, I would use getDependencies otherwise.

I have also considered adding an interface declaring this method. It might be called `SomeDependency` and implemented by JavaAccess, SliceDependency, ModuleDependency, and Dependency. I decided against it only because I was not sure about its name.